packages/api/src/model-input.ts

```diff
@@ -70,10 +68,29 @@ export interface InterventionPeriod {
   reductionPopulationContact: number
 }
 
+export interface Intervention {
+  type: InterventionType
+  intensity?: Intensity
+  reductionPopulationContact?: number
+}
+
 export type ISODate = string
 
 export enum Intensity {
   Mild = 'mild',
   Moderate = 'moderate',
   Aggressive = 'aggressive',
 }
+
+export enum InterventionType {
+  CaseIsolation,
```

Comment:
I _think_ if you change each row to:

    CaseIsolation = 'CaseIsolation',

Then the json schema enum will use strings, not numbers.